cli_tools_e2e_test/go.mod

```diff
@@ -5,13 +5,11 @@ go 1.13
 require (
 	cloud.google.com/go/storage v1.10.0
 	github.com/GoogleCloudPlatform/compute-image-tools/cli_tools v0.0.0-20200813223603-3672ca27e050
+	github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common v0.0.0
 	github.com/GoogleCloudPlatform/compute-image-tools/daisy v0.0.0-20200813213118-5b8ac20eab97
 	github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils v0.0.0-20200813223603-3672ca27e050
 	github.com/aws/aws-sdk-go v1.34.4
-	golang.org/x/net v0.0.0-20200813134508-3edf25e44fcc // indirect
-	golang.org/x/tools v0.0.0-20200813203630-136574234359 // indirect
 	google.golang.org/api v0.30.0
-	google.golang.org/genproto v0.0.0-20200813001606-1ccf2a5ae4fd // indirect
-	google.golang.org/grpc v1.31.0 // indirect
-	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
 )
+
+replace github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common => ./common
```

Comment:
Is this replacement needed in the final code or it's just for your local testing?

Reply:
It is required for the final code; it instructs go's module resolver to use a local directory: https://golang.org/ref/mod#go-mod-file-replace

Specifically, I want the new tests to have compile-time dependency on `cli_tools` in the current code tree. The current modules in `cli_tools_e2e_test` depends on a snapshot of that directory.

Reply:
This is go.mod for cli_tools_e2e_test package and the replacement is for cli_tools_e2e_test/common to common, which is in the same package, so it looks like a no-op to me. What am I missing?

Reply:
I made `common` into a module so that I could depend on it directly, rather than depending on all of the dependencies of the other test suites. See 'cli_tools_e2e_test/common/go.mod'

Reply:
>... rather than depending on all of the dependencies of the other test suites...

Can you please elaborate why is this needed?

Reply:
Tactically, this appears in https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1378, where the feature is landed with the e2e test; this only works since the module in that PR depends on the current source tree of `cli_tools` rather than a pre-pushed snapshot.

More info: The `gce_*` test suites share the [same set of dependencies](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools_e2e_test/go.mod). If the dependencies don't make sense for a particular suite, the developer has two subpar options: (1) modify all test suites to work with the different dependency, (2) modify their test suite to work with the other test suites.

Currently, I'm